pg-migration/src/pg/migration/core.clj

```diff
@@ -166,7 +167,9 @@
   (when-let [[_ id-raw slug-raw direction-raw]
              (re-matches RE_FILE (-> url
                                      .getFile
```

Comment:
The key change: we parse not the entire file path but only the trailing file name, e.g.

`file://path/to/jar-1.2.3/resoures/123.foo.sql` -> `123.foo.sql`